TodoListClient/App.config

```diff
@@ -6,9 +6,9 @@
   <appSettings>
     <add key="ida:AADInstance" value="https://login.microsoftonline.com/{0}" />
     <add key="ida:Tenant" value="[Enter the name of your tenant, e.g. contoso.onmicrosoft.com]" />
-    <add key="ida:ClientId" value="[Enter the Client Id of your application from the portal, e.g. 7de803e2-9k32-4e7a-8335-b77ae40297f0]" />
-    <add key="ida:RedirectUri" value="[Enter the Redirect URI of your application from the portal, e.g. http://TodoListClient]" />
-    <add key="todo:TodoListResourceId" value="[Enter the App Id URI of your application from the portal, e.g. https://contoso.onmicrosoft.com/TodoListService]" />
+    <add key="ida:ClientId" value="[Enter the Application ID (Client ID) of your application from the portal, e.g. 7de803e2-9k32-4e7a-8335-b77ae40297f0]" />
```

Comment:
I would recommend using the same names from our Guided Setups, removing things like ida: - which does not make sense for a lot of people, and change AADInstance to Authority.

See: https://review.docs.microsoft.com/en-us/azure/active-directory/develop/guidedsetups/active-directory-aspnetwebapp-v1?branch=pr-en-us-28192

<add key="ClientId" value="Enter_the_Application_Id_here" />
<add key="RedirectUrl" value="Enter_the_Redirect_Url_here" />
<add key="Tenant" value="common" />
<add key="Authority" value="https://login.microsoftonline.com/{0}" /> 

This is based on feedback from the team for the guided setup. Perhaps we should adopt these names for new samples